packages/core/src/shadow/enum/ShadowMode.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Determines which type of shadows should be used.
+ */
+export enum ShadowMode {
+  /** Disable Shadows. */
+  None,
+  /** Hard Shadows Only. */
+  Hard,
+  /** Cast "soft" shadows (with bi-linear interpolation). */
+  Soft,
+  /** Cast "soft" shadows (with bi-linear PCF filtering). */
+  VerySoft
+}
```

Comment:
- Use `SoftLow` and `SoftHigh`,later you can add `SoftMiddle`, so you will named `VeryVerySoft `😄
- Another thing is you don't need to write comments with `with bi-linear interpolatio`, because this hidden for front developer and when we add SRP，the implementation will change, and the internal implementation of each pipeline is different